doc/source/user_guide/io.rst

```diff
@@ -1305,14 +1305,29 @@ You can elect to skip bad lines:
     0  1  2   3
     1  8  9  10
 
+Or pass a callable function to handle the bad line if ``engine="python"``.
+The bad line will be a list of strings that was split by the ``sep``:
+
+.. code-block:: ipython
```

Comment:
OT: if we can use ipython blocks i think iwould be way better here (this is a general comment)

Reply:
Agreed. Tried changes these to ipython blocks but a lot of errors were raised. Can look into in the future

Reply:
kk great if you can create an issue

Reply:
https://github.com/pandas-dev/pandas/issues/45235